### DISCUSSION TITLE
Trying to get a horizontally mounted phone/cam to show properly in Control AND in Timelapse

### DISCUSSION THREAD

**Discussion by gregreid** [feelfreelinux/octo4a#192] [General]
First of all, THANK YOU SO MUCH for octo4a.  I've repurposed an old Samsung Galaxy S5 for it, and it's working well.  I hard-wired a 5V power supply to its "wireless charging" pins, which is working great.  But I've found one glitch and wonder how to properly report it.  Here?  Well...

I have my phone mounted horizontally.  In the web browser interface settings, the camera supports the various flip horizontally/vertically/rotate90 options, but not so for the timelapse.  It seems that timelapse takes its feed directly from octo4a on the phone, with no "filtering" supported.  

So I found it necessary to use the settings in octo4a on the phone to do the appropriate 90 degree rotation at the source.  So now timelapse is properly oriented.  Great.  But then I needed to use the "rotate 90 degrees counterclockwise" in the web browser webcam settings to get it to show properly oriented.  So now everything (camera, webcam, and timelapse) all show the properly oriented view.  Great, but...

The webcam view in the browser (in the "Control" tab), while properly oriented, is taking up too much screen real-estate -- and overlapping the part of the web screen that has the "Cancel" button for printing, the refresh button for the uploaded file list, and other necessary things.  I have a wide display with plenty of whitespace remaining for a properly sized and located webcam view, but there's no apparent way to resolve this in the various settings.

Thanks for any help,
      Greg

**Comment by alufers** [comment]
See also: #279
